lib/tokenizers.js

```diff
@@ -83,7 +88,17 @@ export async function getCurrent() {
   }
 
   // 2. Get the current tokenizer.
-  return _getCurrentTokenizer();
+  let tokenizer;
+  try {
```

Comment:
Let's move this try over the whole function so that if `deprecateCurrent` throws it will also hit the catch.

Reply:
Done (try/catch expanded).